build/README.md

```diff
@@ -234,6 +234,52 @@ $ make minikube-transfer-image TAG=myimage:0.1
 
 Running end-to-end tests on minikube is done via the `make minikube-test-e2e` target. This target use the same `make test-e2e` but also setup some prerequisites for use with a minikube cluster.
 
+### I Made A Change, Now What?
+
+Awesome! You took your first step to becoming a contributor!
+
+We have several things we can do at this point, some of which you have already seen!
+
+#### Rerun all the tests
+
+Check your code against the current test suite - run `make test` to see if your changes broke any of the current test
+suite.
+
+#### Lint your code
+
+Run `make lint` - this will run the linter over your code (the same as we do in CI), which will look for bad comments,
+errors that haven't been checked, and more. It's a great tool, but can take a while to complete, so you may want to
+run it sparingly.
+
+#### Build and install your changes on a cluster
+
+Assuming you already followed the above section, you should have some kind of test cluster already up, and running
+Agones. To build and install your changes on your cluster, you can repeat the steps you took above - but here's a nice
+single line command:
+
+##### GKE
+
+`make build-images gcloud-auth-docker push install`
```

Comment:
this didn't work for me.
if you don't do any git operations the image label will remain the same and the pull policy of the controller image is "IfNotPresent", so nothing will happen.

Reply:
This should now work, because @Kuqd made it so there is a new annotation that will force the reset on a new install. And if you installed through the `make install` the pull policy should be `Always`. I will confirm though!

Reply:
`Always` means that it will re-pull the image when recreating the controller. The question is: why would the controller be recreated if the spec file is identical?

Reply:
The annotation update _should_ force a redeploy. I will confirm though.

Reply:
FYI, when I'm doing 'kubectl apply -f agones.yaml' I get this output:
```
service "agones-controller-service" unchanged
deployment "agones-controller" unchanged
```

Don't know about helm though...

Reply:
Ah, so we go to "Running a clustom test cluster" section, as I'm not using GKE :)
Anyway, is this a confirmation that `make install` is always recreating the controller?

Reply:
Confirmed! `make install` is always recreating the controller.